test_infra/src/lib.rs

```diff
@@ -589,3 +590,17 @@ pub fn ssh_command_ip(
         timeout,
     )
 }
+
+pub fn exec_host_command_status(command: &str) -> ExitStatus {
+    std::process::Command::new("bash")
+        .args(&["-c", command])
+        .status()
+        .expect(&format!("Expected '{}' to work", command))
```

Comment:
This is only the error from the exec and wait. So the error message is a bit confusing here.

Reply:
Yes it should be more something like `Expected '{}' to run`.